File: core/src/main/java/com/graqr/threshr/model/redsky/product/ReturnPolicy.java
package com.graqr.threshr.model.redsky.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record ReturnPolicy(@JsonProperty("user_type") String userType, @JsonProperty("day_count") Long dayCount) {
}

File: src/main/kotlin/com/graqr/threshr/model/redsky/product/ReturnPolicy.kt
package com.graqr.threshr.model.redsky.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class ReturnPolicy(
    @param:JsonProperty("user_type") val userType: String, @param:JsonProperty("day_count") val dayCount: Long
) {}
